meetup.rb

```diff
@@ -0,0 +1,72 @@
+require 'httparty'
+require 'fileutils'
+require 'colorize'
+require 'dotenv'
+
+# .env
+Dotenv.load
+meetup_api_key = ENV["MEETUP_API_KEY"]
+
+meetup_group_url = "ny-tech"
+# meetup_group_url = "RubyFTW-Fort-Worth-Ruby-User-Group"
+base_uri = "https://api.meetup.com/2/events?key=#{meetup_api_key}&group_urlname=#{meetup_group_url}&sign=true"
+response = HTTParty.get(base_uri)
+body = JSON.parse(response.body)
+
+mu = body["results"][0]
+
+@file_name_date = Time.at(mu["time"]/1000).strftime("%Y-%m-%d")
+@title = mu["name"]
+@description = mu["description"]
+@long_date = Time.at(mu["time"]/1000).strftime("%B %d, %Y")
+@link = mu["event_url"]
+@time = Time.at(mu["time"]/1000).strftime("%I:%M%P")
+@location = mu["venue"]["name"]
+@location_url = "https://www.google.com/maps/search/#{@location}"
+@rsvp_url = mu["event_url"]
+
+@directory = "_upcoming_events"
+@archived_directory = "_archived_events"
+@directory_file_count = Dir[File.join(@directory, '**', '*')].count { |file| File.file?(file) }
+@yes_or_no = "[Y]".green + "Yes" + " [N]".red + "No"
+
+def validate_to_archive
+  answer_to_archive = gets.chomp
+  if answer_to_archive === "Y" || answer_to_archive === "y"
+    puts "Moving files...".light_blue
+    FileUtils.mv Dir.glob("#{@directory}/*"), @archived_directory
```

Comment:
Yes, as of now *all* the files go to `_archived_events`.  Even if you iterate through each files in the future, you would probably still need to manually decided since the data is created into Jekyll rather than just parsing the live data from MeetUp.  For example, I can think of filtering through status of the event but that will be challenging since you are basically copying and pasting data into Jekyll file oppose to live data.  Any thoughts?